### ISSUE TITLE
bug elErrorContainer (maybe) in version 4.4.3

### ISSUE THREAD

**Issue by b4p3p** [kartik-v/bootstrap-fileinput#1066]
Hi!
in the version 4.4.3, i have found a little bug.
when my server sends an error message, the elErrorContainer doesn't show the error and in console i have this error:

`ERROR TypeError: Cannot read property 'find' of undefined
    at FileInput._refreshUploadButton (eval at webpackJsonp.../../../../script-loader/addScript.js.module.exports (addScript.js:9), <anonymous>:2932:43)
    at FileInput._setPreviewError (eval at webpackJsonp.../../../../script-loader/addScript.js.module.exports (addScript.js:9), <anonymous>:2929:18)
    at eval (eval at webpackJsonp.../../../../script-loader/addScript.js.module.exports (addScript.js:9), <anonymous>:2074:26)
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:425)
    at Object.onInvokeTask (core.es5.js:3881)
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:424)
    at Zone.webpackJsonp.../../../../zone.js/dist/zone.js.Zone.runTask (zone.js:192)
    at webpackJsonp.../../../../zone.js/dist/zone.js.ZoneTask.invokeTask (zone.js:499)
    at ZoneTask.invoke (zone.js:488)
    at timer (zone.js:1896)`

with version 4.4.2 it works fine

I'm using angular 4 with bootstrap

Thank's for all

**Comment by kartik-v** [comment]
Are you setting [removeFromPreviewOnError](http://plugins.krajee.com/file-input#option-removefrompreviewonerror) to `true` for your case?

**Comment by b4p3p** [comment]
I tried it right now after upgrade.

same error with version 4.4.3 with removeFromPreviewOnError.

however...

Version 4.4.2 with argument removeFromPreviewOnError: true, i have this error

```
core.es5.js:1020 ERROR TypeError: Cannot read property 'remove' of undefined
    at FileInput._setPreviewError (eval at webpackJsonp.../../../../script-loader/addScript.js.module.exports (addScript.js:9), <anonymous>:2898:23)
    at eval (eval at webpackJsonp.../../../../script-loader/addScript.js.module.exports (addScript.js:9), <anonymous>:2068:26)
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:425)
    at Object.onInvokeTask (core.es5.js:3881)
    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:424)
    at Zone.webpackJsonp.../../../../zone.js/dist/zone.js.Zone.runTask (zone.js:192)
    at webpackJsonp.../../../../zone.js/dist/zone.js.ZoneTask.invokeTask (zone.js:499)
    at ZoneTask.invoke (zone.js:488)
    at timer (zone.js:1896)
```

Version 4.4.2 without argument removeFromPreviewOnError works fine

this is my config

```javascript
this.txtFileLibri = this.guiService.initBootstrapFileinput($("#file_libri"), {
      removeFromPreviewOnError: true,
      elErrorContainer:"#file_libri_error",
      uploadUrl:"/api/libri/import",
      allowedFileExtensions: ["xlsx"],
      showUpload: false,
      uploadExtraData: function () {
        return {
          scuola: self.dataService.scuolaSelezionata.getID(),
          nome_classe: self.nome_classe  }
      },
      onError: function (err) {
        self.messageService.error("Impossibile importare gli elementi", err)
      },
      onSuccess:function (ris) {
        self.ris_upload_anagrafiche = ris;
      },
      onChange:()=>{ self.hasFileLibri = true; },
      onFileClear:()=>{ self.hasFileLibri = false; },
    });
```

**Comment by kartik-v** [comment]
Ok this will be fixed... there are some significant updates coming up shortly with enhancements for Bootstrap v4.x (#1065).

**Comment by kartik-v** [comment]
Resolved via [updates to release v4.4.4](https://github.com/kartik-v/bootstrap-fileinput/commit/05c2ca1631ccec305919af5bf02a5692dc960e3e).
